### ISSUE TITLE
Obsidian plugin

### ISSUE THREAD

**Issue by maxime1992** [BaldissaraMatheus/Tasks.md#100]
Hello, this looks neat !
I'd love to be able to use it in my obsidian vault directly, any chance you would consider publishing a plugin ?

**Comment by BaldissaraMatheus** [comment]
I don't think you need a plugin for that, you already can sync your tasks with obsidian. Since it uses the OS filesystem, you just have to have both apps using the same directory. I included a short step-by-step on #49, please read it and let me know if it helps

**Comment by maxime1992** [comment]
> I don't think you need a plugin for that,

Good to know there's at least something I could use. But I can still think of a really good use case for the plugin. Essentially people using obsidian would be able to run the app directly from obsidian, avoiding the need to have an extra app and URL to access. Essentially centralising everything in Obsidian. 

To give you a rough idea of the interest in that area, here are some Obsidian plugins for Kanban that are doing something similar (but the one I currently use doesn't have tags filtering...)

![image](https://github.com/user-attachments/assets/2ec95165-88fa-4e6c-b9b3-3310c1eecba0)

There's quite a lot of download on those apps. 

I also understand if you're not interested in building that as it differs from the original idea, feel free to just close the issue if that's the case :)

**Comment by BaldissaraMatheus** [comment]
Got it, I included a enhancement label and I'll consider doing it later

**Comment by maxime1992** [comment]
Awesome, thanks.

Also, a really nice thing that'd come out of that is offline support. As obsidian is offline first, once the plugin is downloaded to obsidian we'd still be able to edit everything from the app :tada:
